src/test/java/uk/gov/companieshouse/ordernotification/messageproducer/MessageFactoryTest.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.companieshouse.ordernotification.messageproducer;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.kafka.test.context.EmbeddedKafka;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.TestPropertySource;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.utility.DockerImageName;
+import uk.gov.companieshouse.ordernotification.emailsender.EmailSend;
+import uk.gov.companieshouse.ordernotification.logging.LoggingUtils;
+import uk.gov.companieshouse.kafka.deserialization.DeserializerFactory;
+import uk.gov.companieshouse.kafka.message.Message;
+import uk.gov.companieshouse.kafka.serialization.AvroSerializer;
+import uk.gov.companieshouse.kafka.serialization.SerializerFactory;
+
+@SpringBootTest
+@DirtiesContext
+@EmbeddedKafka
+@TestPropertySource(locations = "classpath:application-stubbed-empty.properties")
+class MessageFactoryTest {
+    @Autowired
+    private SerializerFactory serializerFactory;
+    @Autowired
+    private DeserializerFactory deserializerFactory;
+    @Autowired
+    private LoggingUtils loggingUtils;
+
+    private static KafkaContainer kafkaContainer;
+
+    private static final String APP_ID = "App Id";
+    private static final String EMAIL_DATA = "Message content";
+    private static final String EMAIL_ADDR = "someone@example.com";
+    private static final String MSG_ID = "Message Id";
+    private static final String MSG_TYPE = "Message type";
+    private static final String ORDER_REF = "ORD-ABC-123";
+    private static final String CREATED_AT = "2020-08-25T09:27:09.519+01:00";
+
+    private static final String TOPIC = "email-send";
```

Comment:
Place above constants inside TestConstants.java

Reply:
Resolved in latest commit